lib/string/similarity.rb

```diff
@@ -94,13 +95,32 @@ def self.base_case?(str1, str2)
   end
 
   # create a vector from +str+
+  # keys have a special format:
+  #     '[left padding, right padding, "string"]'
   #
   # @example
-  #     v1 = vector('hello') # => {"h"=>1, "e"=>1, "l"=>2, "o"=>1}
-  #     v1["x"] # => 0
-  def self.vector(str)
+  #     v1 = vector('aba', 1) # => {'[0, 0, "a"]' => 2, '[0, 0, "b"]' => 1}
+  #     v1['[0, 0, "x"]'] # => 0
+  # @example
+  #     vector('abacaba', 2) # => {
+  #       #  '[1, 0, "a"]' => 1,
+  #       #  '[0, 0, "ab"]' => 2,
+  #       #  '[0, 0, "ba"]' => 2,
+  #       #  '[0, 0, "ac"]' => 1,
```

Comment:
If I understood this correctly, then "ca" is missing from the example?

```suggestion
  #       #  '[0, 0, "ac"]' => 1,
  #       #  '[0, 0, "ca"]' => 1
```

Reply:
Yes, right. It is present in spec https://github.com/mhutter/string-similarity/pull/8/files/944e8a9e4d065b819b349284ae387313fdcc5fc5#diff-1e6aa52f043868d4f98f17b3263127bbR52 but I think I missed it here.